packages/notifi-react-card/lib/components/subscription/EventTypeBroadcastRow.tsx

```diff
@@ -75,6 +77,10 @@ export const EventTypeBroadcastRow: React.FC<EventTypeBroadcastRowProps> = ({
     >
       <div className={clsx('EventTypeBroadcastRow__label', classNames?.label)}>
         {config.name}
+        <NotifiTooltip
```

Comment:
does this show up even if there isn't content being passed in?

Reply:
ah, good catch, updated